.github/workflows/publish-docker-tannsi-relay.yml

```diff
@@ -0,0 +1,75 @@
+name: Publish Docker
```

Comment:
Mention dancelight here, to avoid confusion with the other "Publish Docker" job

Reply:
Sure thing updated here [e4e9ce4](https://github.com/moondance-labs/tanssi/pull/696/commits/e4e9ce405401f17909d457da806b8115c4a7f7ed)